src/compute/src/logging/initialize.rs

```diff
@@ -125,7 +125,8 @@ impl<A: Allocate + 'static> LoggingContext<'_, A> {
         traces
             .into_iter()
             .map(|(log, (trace, token))| {
-                let bundle = TraceBundle::new(trace, errs.clone()).with_drop(token);
+                let bundle = TraceBundle::new(SpecializedTraceHandle::RowRow(trace), errs.clone())
+                    .with_drop(token);
```

Comment:
This seems fragile in the log run. It'd break the invariant when we'd introduce an introspection arrangement that would use specialization.

Reply:
This is a good point, which I captured in issue https://github.com/MaterializeInc/materialize/issues/22398 so that we can address as follow-up work. I also added a comment to the code pointing to the issue.